src/web/demo.rkt

```diff
@@ -390,31 +484,29 @@
     (lambda (post-data)
       (define formula-str (hash-ref post-data 'formula))
       (define formula (read-syntax 'web (open-input-string formula-str)))
-      (define hash (sha1 (open-input-string formula-str)))
-      ;; Hmm how should I pass seed?
-      ;; is it ok to use the seed set in the thread?
       (define seed (hash-ref post-data 'seed))
-      ;; Is this ok because we are multithreaded now?
-      (set-seed! seed)
-      (semaphore-wait (run-sample hash formula))
+      (define hash (sha1 (open-input-string formula-str)))
```

Comment:
I think I should hash in the "command" which would be `'sample` in this case to distigusish between different commands for the same formula.

Reply:
Agreed